metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -700,6 +711,16 @@ class TableauSourceReport(
         default_factory=(lambda: defaultdict(int))
     )
 
+    get_connection_objects_query_counter: Dict[str, int] = dataclass_field(
+        default_factory=(lambda: defaultdict(int))
+    )
+    get_connection_objects_query_filter_pages_counter: Dict[str, int] = dataclass_field(
+        default_factory=(lambda: defaultdict(int))
+    )
+    get_connection_objects_query_total_pages_counter: Dict[str, int] = dataclass_field(
```

Comment:
these names are super verbose - can we simplify

Reply:
addressed in commit ab10931